app/javascript/pipeline-graph/index.js

```diff
@@ -409,7 +415,11 @@ fetch('pipelines.json').then((response) => {
     // Get filter from url
     const url = new URL(window.location.href)
     const filter = url.searchParams.get('filter')
-    filterField.value = filter // set before rendering the graph for the users benefit
+    const group = url.searchParams.get('group')
+    const showSubPipelines = group === 'true' ? false : group === 'false' ? true : undefined // convert string to boolean
```

Comment:
I had to look twice at this 😆 
I would maybe do this in two lines - one to convert string to boolean, and one to reverse the sign - or, update your comment to say you're also switching true to false and false to true!

Reply:
Thanks, I think that's clearer 👍